src/commonStyles/variables.scss

```diff
@@ -0,0 +1,14 @@
+:root {
+    --background-footer-main-color: #59ABB0;
```

Comment:
sass variables are defined in other way than css ones: https://sass-scss.ru/documentation/sassscript/peremennie/